Project/jobportal/index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Job Portal</title>
+    <link rel="stylesheet" href="index.css">
+</head>
+
+<body>
+    <header class="nav-header">
+        <h1>Job Portal</h1>
+        <nav>
+            <ul>
+                <li><a href="#">Home</a></li>
```

Comment:
in ul, li , ol how can we provide different type of bulletins, like roman(i, II), -, a, A, 1, and so on..